public/javascript/ide.js

```diff
@@ -336,7 +340,13 @@ function createVoiceRecorder() {
          }
 
          chunks = [];
-			}
+		 
+		 //reset mediaRecorder settings
+		 recorder.onclick = function() {
+		 recorder.value = "Stop";
+         mediaRecorder.start();
+		 }
```

Comment:
Should we fix some of the formatting here? Just so things are readable when we make adjustments later.

Reply:
Haha. That's definitely not what it looked like on my screen before the push. Oh vim!